src/compat/text.rs

```diff
@@ -161,12 +161,14 @@ where
     for line in reader.lines() {
         let line =
             line.map_err(|e| ErrorKind::io_error("Cannot read line from embedding file", e))?;
-        let mut parts = line.split_whitespace();
+        let mut parts = line
+            .split(|c| c == ' ' || c == '\t')
+            .filter(|c| !c.is_empty());
 
         let word = parts
             .next()
             .ok_or_else(|| ErrorKind::Format(String::from("Spurious empty line")))?
-            .trim();
+            .trim_matches(|c| c == ' ' || c == '\t');
```

Comment:
Would

`.trim_matches(char::is_ascii_whitespace)`

work?

Reply:
Pushed out the changes, trimming and splitting now happens based on `is_ascii_whitespace`.